language-server/modules/langserver-core/src/test/resources/completion/annotation_decl/source/source21.bal

```diff
@@ -0,0 +1,10 @@
+
+import ballerina/module1;
+
+annotation testAnnotation on class;
+
+
+
```

Comment:
Are these lines required? line 1, 6,  and 7?

Reply:
This is added intentionally